### ISSUE TITLE
NTR: Renal biopsy: large nuclei in renal tubules (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9309]
New term request Renal biopsy: large nuclei in renal tubules (elthat_130430035151): Karyomegaly  UMLS: 3553775

**Comment by pnrobinson** [comment]
@MickeySegal

Hi, @avizrosenberg @yongqunh  -- could you help with this item?
I am seeing Figures 1 and 2 here, but there are multiple other abnormalities and I am not sure of the best way to classify them:
https://www.ajkd.org/article/S0272-6386(16)30114-7/fulltext

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000429
- label: Renal tubular karyomegaly
- parents: [Abnormal renal tubule morphology (HP:0000091)](https://hpo.jax.org/app/browse/term/HP:0000091)
- pmid: PMID:15311851
- def: Marked enlargement of nuclei observed predominantly in the proximal convoluted tubule cells.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9309
- synonyms:
